src/app/core/json-patch/json-patch-operations.service.ts

```diff
@@ -144,7 +145,7 @@ export abstract class JsonPatchOperationsService<ResponseDefinitionDomain, Patch
    *    instance of PatchRequestDefinition
    */
   protected getRequestInstance(uuid: string, href: string, body?: any): PatchRequestDefinition {
-    return new this.patchRequestConstructor(uuid, href, body);
+    return new this.patchRequestConstructor(uuid, new URLCombiner(href, '?embed=item').toString(), body);
```

Comment:
@artlowel : I'm a little worried that this change is in the generic `JsonPatchOperationsService` as it seems *very specific* to the `SubmissionJsonPatchOperationsService`.  Is there a way to move this to `SubmissionJsonPatchOperationsService to provide the same behavior?

Right now, this seems to work because we don't use the `JsonPatchOperationsService` anywhere except the Item Submission form.  But, since it's a generic class, it seems like it **could be used elsewhere**.  Whenever it is used elsewhere, this "?embed=item" may cause odd issues if we are no longer PATCHing item objects.

So, while I'm not entirely against this change... I wanted to point out that it looks odd because it's modifying a generic service with code that seems specific to the Submission form.

If you don't see a way around this, we can leave this as-is...because the changes to the specs should "protect" us.  But, I wanted to point out this seems to make `json-patch-operations.service.ts` even more specific to Item Submissions.

Reply:
In practice the `JsonPatchOperationsService` is only used for the patches in the submission. There's no need for it to be abstract. It was part of the original submission PR that didn't make use of dataservices or the cache. I wouldn't recommend its use anywhere else.

However it's easy to move the fix to the SubmissionJsonPatchOperationsService so I'll do it

Reply:
Thanks for the explanation @artlowel .  It just seemed like the SubmissionJsonPatchOperationsService is the more appropriate location for this code, so I appreciate moving it there.  Perhaps at some point we need to simply merge the JsonPatchOperationsService all into the SubmissionJsonPatchOperationsService to make it clear that it is *only to be used* for Submission forms.  Currently, the code structure implies JsonPatchOperationsService is generic, when it sounds like it really isn't. That can happen in some future PR refactor though.